FirebaseSessions/Tests/TestApp/Podfile

```diff
@@ -0,0 +1,28 @@
+# Uncomment the next line to define a global platform for your project
+# platform :ios, '9.0'
+
+def shared_pods
+  pod 'FirebaseCore', :path => '../../../'
+  pod 'FirebaseInstallations', :path => '../../../'
+  pod 'FirebaseCoreInternal', :path => '../../../'
```

Comment:
Curious if we need these internal pods? If we don't use (which I presume it is), can we remove them from the list of dependencies?

Reply:
Infact, I'm thinking if we can depend just on FirebaseSessions here and all the other dependencies should naturally be brought in.

Reply:
You need all the recursive dependencies. Otherwise, you'll get the latest release versions instead of the ones from the repo.

Reply:
Aah! Got it. Makes sense. Its been a while, got it.